email/emails/registration-confirm.tsx

```diff
@@ -40,9 +41,18 @@ export default function RegistrationConfirmEmail() {
         />
       </Text>
       <Text>
+        <If condition="${isSignup}">
+          <If.Then>
+            <LocalizedText
+              keyName="registration-welcome-text"
+              defaultValue="Welcome and thank you for creating an account!"
+            />{' '}
+          </If.Then>
+          <If.Else></If.Else>
```

Comment:
Nit: `If.Else` isn't strictly mandatory; looking back at the HACKING.md note I see where it comes from but I more or less wanted to express that bare JSX as children of `If` isn't great because it's hard to tell what part of an `if-then-else` we're working with.

Empty else doesn't hurt, but omitting it is fine imho.